### DISCUSSION TITLE
打包本地自定义html

### DISCUSSION THREAD

**Discussion by itrookie** [tw93/Pake#149] [Ideas]
可以实现吗？

**Comment by tw93** [comment]
试试将html放cdn试试

**Reply by d93k9f34f934jr** [reply]
比如说我的nas搭建了很多自定义页面，如果全部打包成桌面app感觉会很酷

**Reply by Tlntin** [reply]
dev分支已支持，即将合并到pake-cli命令行打包工具内。
下面是https://github.com/mayswind/AriaNg/releases 页面下载的纯index.html打包的app示例。
![image](https://user-images.githubusercontent.com/28218658/209783387-01eb9941-2bb6-4b8b-a13a-7fc73aa08f88.png)

**Comment by Tlntin** [comment]
> 可以实现吗？

可以试试了。
目前应该基本Ok了。
使用方法：
1. 卸载当前版本。
```bash
npm uninstall pake-cli -g
```
2. 安装最新dev版本。
```bash
npm install https://github.com/tw93/Pake.git#dev -g
```
3. 打包当前目录的index.html文件（暂时只支持单个文件）。
```bash
pake ./index.html
```
更多使用说明可以看dev分支的[文档](https://github.com/tw93/Pake/tree/dev/bin#%E7%94%A8%E6%B3%95)。
